person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/DataPostProcessor.java

```diff
@@ -82,9 +82,11 @@ public <T extends PersonExtendedProps> void processPersonRace(String race, T pf)
     public <T extends PersonExtendedProps> void processPersonTelephone(String telephone, T pf) {
         if (!ObjectUtils.isEmpty(telephone)) {
             Function<String, T> personPhoneFn =
-                    (useCd) -> Arrays.stream(utilHelper.deserializePayload(telephone, Phone[].class))
-                            .filter(phone -> StringUtils.hasText(phone.getUseCd())
-                                    && phone.getUseCd().equalsIgnoreCase(useCd))
+                    (code) -> Arrays.stream(utilHelper.deserializePayload(telephone, Phone[].class))
+                            .filter(phone -> (StringUtils.hasText(phone.getUseCd())
+                                    && phone.getUseCd().equalsIgnoreCase(code)) ||
+                                    (StringUtils.hasText(phone.getCd())
```

Comment:
@upasanapattnaik-eq Did you already port in this changes in your pR?

Reply:
Yup! They should be in a PR in the other repo.